Dockerfile

```diff
@@ -3,3 +3,4 @@ COPY build/. /usr/share/nginx/html/
 
 VOLUME /etc/nginx/conf.d
 VOLUME /usr/share/nginx/html/public/cim
+VOLUME /usr/share/nginx/html/public/conf
```

Comment:
Cannot really judge on this one :)

Reply:
This way we can override the configuration on deployed if needed.
So if Alliander wants a real specific one we can do it without changing CoMPAS.

Reply:
Nice